backend/pom.xml

```diff
@@ -45,6 +45,12 @@
             <artifactId>jasypt-spring-boot-starter</artifactId>
             <version>3.0.3</version>
         </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>1.18.24</version>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
You're right, I'll change it so we stay consentient. The main reason I used it for, was that it provides @Builder, and @Value - which is basically same as Record, that I couldn't use because of Java version. But as I'm using those literally once it makes sense to get rid of them.